state_management/csharp/http/README.md

```diff
@@ -22,10 +24,39 @@ name: Install Dotnet dependencies
 cd ./order-processor
 dotnet restore
 dotnet build
+cd ..
 ```
 
 <!-- END_STEP -->
-2. Run the Dotnet service app with Dapr: 
+
+2. Run the Dotnet service app with Dapr:
+
+<!-- STEP
+name: Run order-processor service
+expected_stdout_lines:
+  - '== APP - order-processor == Getting Order: {"orderId":1}'
+  - '== APP - order-processor == Getting Order: {"orderId":2}'
+  - "Exited App successfully"
+expected_stderr_lines:
+output_match_mode: substring
+background: true
+sleep: 15
```

Comment:
if testing is flakey or slow consider a timeout of 30 here

Reply:
Sleep is working well, so increased the sleep time to 60 secs